colabs/intro/Report_API_Quickstart.ipynb

```diff
@@ -1,1599 +1,1799 @@
+        "            wr.PCColumn(metric=\"c::dropout\"),\n",
+        "            wr.PCColumn(metric=\"c::epochs\"),\n",
+        "            wr.PCColumn(metric=\"c::fc_layer_size\"),\n",
+        "            wr.PCColumn(metric=\"c::learning_rate\"),\n",
+        "            wr.PCColumn(metric=\"c::optimizer\"),\n",
+        "            wr.PCColumn(metric=\"loss\"),\n",
+        "        ]\n",
+        "    )\n",
+        "]\n"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "fSBotQKy8w8G"
+      },
+      "source": [
+        "5. Save the report"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "zMUv2i6_8w8G"
+      },
+      "outputs": [],
+      "source": [
+        "report.save()"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "aOct2pR28w8G"
+      },
+      "source": [
+        "### The same thing all-in-one"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "psBEGgwF8w8G"
+      },
+      "outputs": [],
+      "source": [
+        "report = wr.Report(\n",
+        "    project=PROJECT,\n",
+        "    title=\"Parallel Coordinates Example (all-in-one)\",\n",
+        "    description=\"Using the pytorch sweeps demo (same as the other one but written in one expression)\"\n",
+        "    blocks=[\n",
+        "        wr.PanelGrid(\n",
+        "            runsets=[wr.Runset(project=\"pytorch-sweeps-demo\")],\n",
+        "            panels=[\n",
+        "                wr.ParallelCoordinatesPlot(\n",
+        "                    columns=[\n",
+        "                        wr.PCColumn(metric='c::batch_size'),\n",
+        "                        wr.PCColumn(metric='c::dropout'),\n",
+        "                        wr.PCColumn(metric='c::epochs'),\n",
+        "                        wr.PCColumn(metric='c::fc_layer_size'),\n",
+        "                        wr.PCColumn(metric='c::learning_rate'),\n",
+        "                        wr.PCColumn(metric='c::optimizer'),\n",
+        "                        wr.PCColumn(metric='loss'),\n",
+        "                    ]\n",
+        "                )\n",
+        "            ],\n",
+        "        )\n",
+        "    ],\n",
+        ")"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "EaV2bZDM8w8G"
+      },
+      "outputs": [],
+      "source": [
+        "report.save()"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "wbFXPPEs8w8G"
+      },
+      "source": [
+        "## I tried mutating an object in list but it didn't work!\n",
+        "tl;dr: It should always work if you assign a value to the attribute instead of mutating.  If you really need to mutate, do it before assignment.\n",
```

Comment:
Remove "TL;DR".  Not all customers will know what that means (especially ESL folks).